src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -797,8 +791,31 @@ HRESULT CorProfiler::GenerateVoidILStartupMethod(const ModuleID module_id,
     return hr;
   }
 
+#ifdef _WIN32
+  WSTRING native_profiler_file = "DATADOG.TRACE.CLRPROFILER.NATIVE.DLL"_W;
+#else // _WIN32
+
+#ifdef BIT64
+  WSTRING native_profiler_file = GetEnvironmentValue("CORECLR_PROFILER_PATH_64"_W);
+  Debug("GenerateVoidILStartupMethod: Linux: CORECLR_PROFILER_PATH_64 defined as: ", native_profiler_file);
+  if (native_profiler_file == ""_W) {
+    native_profiler_file = GetEnvironmentValue("CORECLR_PROFILER_PATH"_W);
+    Debug("GenerateVoidILStartupMethod: Linux: CORECLR_PROFILER_PATH defined as: ", native_profiler_file);
+  }
+#else // BIT64
```

Comment:
Should this comment say this else is for 32 bit?

Reply:
I was following the convention I observed in CoreCLR where each comment describes the original `if` condition. I'm open to changing it though

Reply:
Ah, hmm... that's an interesting convention.... I'm happy to adopt it.